test/functional/test_runner.py

```diff
@@ -215,6 +215,7 @@
     'rpc_blockchain.py',
     'rpc_deprecated.py',
     'wallet_disable.py',
+    'wallet_change_address.py',
```

Comment:
This should have `--legacy-wallet` and `--descriptors` variants as the value of `hdkeypath` is different depending on the wallet type.

Also, this test fails with `--legacy-wallet`.

Reply:
Thanks! Added the variants. `import_descriptors` is still failing, will fix that next.